includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -229,300 +262,132 @@ public function setup_wizard_content() {
-					</td>
-				</tr>
-				<tr class="tax-rates" style="display: none;">
-					<td colspan="2">
-						<p><?php printf( __( 'The following tax rates will be imported automatically for you. You can read more about taxes in <a href="%s" target="_blank">our documentation</a>.', 'woocommerce' ), 'https://docs.woocommerce.com/document/setting-up-taxes-in-woocommerce/' ); ?></p>
-						<div class="importing-tax-rates">
-							<table class="tax-rates">
-								<thead>
-								<tr>
-									<th><?php esc_html_e( 'Country', 'woocommerce' ); ?></th>
-									<th><?php esc_html_e( 'State', 'woocommerce' ); ?></th>
-									<th><?php esc_html_e( 'Rate (%)', 'woocommerce' ); ?></th>
-									<th><?php esc_html_e( 'Name', 'woocommerce' ); ?></th>
-								</tr>
-								</thead>
-								<tbody></tbody>
-							</table>
-						</div>
-						<p class="description"><?php printf( __( 'You may need to add/edit rates based on your products or business location which can be done from the <a href="%s" target="_blank">tax settings</a> screen. If in doubt, speak to an accountant.', 'woocommerce' ), esc_url( admin_url( 'admin.php?page=wc-settings&tab=tax' ) ) ); ?></p>
-					</td>
-				</tr>
-			</table>
+			<p><?php esc_html_e( "This quick setup wizard will help you configure the basic settings, and shouldn't take longer than five minutes. To get started, we need to know a few details about your store.", 'woocommerce' ); ?></p>
+			<div>
+				<label><?php esc_html_e( 'Where is your store based?', 'woocommerce' ); ?></label>
+			</div>
+			<div>
+				<label for="store_address"><?php esc_html_e( 'Address', 'woocommerce' ); ?></label>
+				<input type="text" id="store_address" name="store_address" style="width:100%;" required value="<?php echo esc_attr( $address ); ?>" />
+				<label for="store_address_2"><?php esc_html_e( 'Address line 2', 'woocommerce' ); ?></label>
+				<input type="text" id="store_address_2" name="store_address_2" style="width:100%;" value="<?php echo esc_attr( $address_2 ); ?>" />
+			</div>
+			<div style="width:33%; float:left;">
+				<label for="store_city"><?php esc_html_e( 'City', 'woocommerce' ); ?></label>
+				<input type="text" id="store_city" name="store_city" required value="<?php echo esc_attr( $city ); ?>" />
+			</div>
+			<div style="width:33%; float:left;">
+				<label for="store_country_state"><?php esc_html_e( 'Country / State', 'woocommerce' ); ?></label>
+				<select id="store_country_state" name="store_country_state" style="width:100%;" required data-placeholder="<?php esc_attr_e( 'Choose a country&hellip;', 'woocommerce' ); ?>" aria-label="<?php esc_attr_e( 'Country', 'woocommerce' ) ?>" class="wc-enhanced-select">
+					<?php WC()->countries->country_dropdown_options( $country, $state ); ?>
```

Comment:
There's a huge map returned by `WC_Countries->get_country_locale()` that tells you how things are named for each country (for example, in some countries `states` are called `provinces`, in USA the postcode is called `ZIP`, etc) and which parts are required/optional for a complete address.

Not super urgent, but we should use that data.